### ISSUE TITLE
ShiftIt doesn't work when another app is accessing microphone on MacOS Monterey

### ISSUE THREAD

**Issue by mohamednaguib** [fikovnik/ShiftIt#328]
Hi all,

I tried this with several applications to make sure that I've this right but it seems that whenever another application is accessing the microphone, ShiftIt shortcuts doesn't work at all. 
I am not sure if it's something related to accessibility bug on the MacOS itself or some weird bug in ShiftIt.

Can someone investigate this further?

**Comment by CocaineLee** [comment]
Same, using the zoom with audio on, the  shiftlt shortcut won't work

**Comment by watermadeline** [comment]
Yes definitely still an issue. Was first reported as just an issue when Zoom was running 6 MONTHS AGO. You're description is more accurate. Would be nice if developers would get on it

**Comment by KazukiOhta** [comment]
This is nice issue, actually, I have the same problem.

**Comment by rca** [comment]
There's [another issue](https://github.com/fikovnik/ShiftIt/issues/320) that suggests migrating over to [Hammerspoon](http://www.hammerspoon.org/) with the [ShiftIt spoon](https://github.com/peterklijn/hammerspoon-shiftit).

After a bunch of conference calls that blocked shortcuts, I finally made the jump.  It's not identical, but close enough to be a good viable and supported solution.

**Comment by matanmarciano** [comment]
Any news regarding this issue?

**Comment by skeshmiri** [comment]
Debug log outputs this

`NSError stack trace:
org.shiftitapp.shifit.error:20103 - Unable to get active window
  Caused-by: org.shiftitapp.shifit.error:20101 - Unable to find focused window owner
`

**Comment by EHLOVader** [comment]
This was still a problem, just realized after switching to my personal computer as a primary machine that I hadn't installed hammerspoon on it like I did my work one. 

Any news on it, or idea what would help move this forward?

**Comment by wsfpeterlai** [comment]
can anyone assist with shift it update ?
?
